src/test/java/seedu/address/model/appointment/AppointmentTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.model.appointment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.model.appointment.Appointment.DATE_FORMAT;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import org.junit.jupiter.api.Test;
+
+
+class AppointmentTest {
+
+    @Test
+    public void isValidDate_nullDate_returnsTrue() {
+        assertTrue(Appointment.isValidDate(null));
+    }
+
+    @Test
+    public void isValidDate_validDate_returnsTrue() {
+        assertTrue(Appointment.isValidDate("01-01-2020"));
+    }
+
+    @Test
+    public void isValidDate_invalidDate_returnsFalse() {
+        assertFalse(Appointment.isValidDate("01-01-2020 12:00"));
```

Comment:
Perhaps a better way to write an invalid date would be "99-99-9999" instead of a string of valid dates + a time. You could always add more `assertFalse` into this same test method `public void isValidDate_invalidDate_returnsFalse()`
as it increases the EP testing.

Reply:
Good catch. Thanks!